### DISCUSSION TITLE
InlineCompletion unclear (when they show up, what's filterText)

### DISCUSSION THREAD

**Discussion by yitao-mutable** [microsoft/vscode-discussions#49] [Extension Development QnA] [answered]
I'm not sure what does `filterText` do and how should I use it even after reading the docstring
![image](https://user-images.githubusercontent.com/106548387/176755741-7039ee6f-9b19-4fb4-ad84-57dacc849bed.png)

Also, I notice the inlineCompletion doesn't show up although the `InlineCompletionItem[]` is returned

**Comment by isidorn** [comment]
@hediet can you please help here?

**Comment by burkeholland** [comment]
Did you find an answer to your question? The doc string looks descriptive. Can you be more specific about what you aren't sure about?

**Reply by yitao-mutable** [reply]
Hi, after 1 year, I'm still aren't sure what factors cause the inline suggestions not showing up despite a non-empty inline suggest array. For example, I noticed different "inline suggestion providers" conflict.

**Reply by hediet** [reply, marked as answer]
Can you reproduce in the [monaco editor](https://microsoft.github.io/monaco-editor/playground.html?source=v0.37.1#XQAAAAK-CAAAAAAAAABBqQkHQ5NjdMjwa-jY7SIQ9S7DNlzs5W-mwj0fe1ZCDRFc9ws9XQE0SJE1jc2VKxhaLFIw9vEWSxW3yscxEpzY6fGetpg8d5BCIdogK6CBmgyKHIOVjH6uO-oA-RA-rcpQdGBlgwMT7NDeGjyzFlJIBUTSIWue6aslVAGOUIo1a7VOvbFeZadLjqSIjynWbkrDNAyZ7f44qIQO8vYvJ604Gw0sw9IexIOLeAbgbun7FbykksyIw1EglcmhMn5rEDj5hwgWUEtfmbdCzOUZozB3-PEoYxNAMiBZ-xs7MpmTQ7KlGGusnsFR1xH0PRDzdplekDJ8GkFpeZsoIHmaNHrVZ-v8pr9wKH2dei_6GjKtG9mTwm9B5C70Pa2vzNqvBR6_XeLc1UK_skAry986T0WucTQAJDWwLIA2ffErYil2cuy0rPRZtmBkOQCp7cvgV3yS0UV8_ut_3bIGKM7ulSKhSdvsHDHbFDdX4TNxvVurB_oPqJy9Myi1D3CkYa7lJP-9hi9yKTdeqSXCsMXtf6qx4LspwipFUcS2i3K_9iYZDAI7XTfn0MEYljFEh9bl2ra01Azx7Jo0Eb9GYMHm5u2Zx1fm0pj1o3pPMnvDdIB_YQqIKOALyaPMOSnXAxjPxDTTN-ipBA-ih__8id9iEtAdOUOrYaumIs9wY2qEBn-3XVRxPgyj-7THLTZER5lLH4VuJ5wxXOOnWBEFbJnWta77XGDuk3RK-6dGdAsEpb6YfSHItjqPQLKs51Ye4wesuSUeo7_uoaIPE9LQbMDsuQ1wFc89JqrSlBmRnirG1pBqFHZHOaCjKLCW9_vUcT45cTtLKSjZxXwjsimptBKy8QG4McB1IFaZNKOcdR2Tz2QNvgQHhBkv5vHomDt8WriOhyYd2zzNugQgKR7Q0TbUeguJ1twQCSGkQAm1MZ97NbgtstlDhd0iM7rzkieRuwpQAemBoDrf4gTQnfDi__4QBiktLmIw_3giWL0amYjaQtU4VkWyjqNzKM2H6hncgTOhBdHqPHRkxgIlJmLx00iiTaQicX_tbhC7o_-Kg9UH5CcnkpTVAeUhuHU1vXzi9ZNa4HnXJlFOm-gjwhOPnY0kkyeO5BJW5NTyD0DYNSDIPBQkPduMtYqG3Rt4i90p7YtNZr_weQzjq_k_uo4Q3tkcfa1pCbLXoUPbXX-eHvq7hH5T1VCSWoD_0I676wfms87CcWHNDs6RwFL4aPVkXVB3tlAon0UcDrKfIqAsbb9SJy3iatw0wM9bVtwQqKZuk80IJJvsovaGhyc4KuUVaPOkyi0ltO1iFYpeeBIROdvYu-Z5VaClJHROeJQ7MPRS8YHBlvP7_RsPRg)?

**Reply by yitao-mutable** [reply]
It's hard to reproduce coz my case is specific to VS Code extension. I just want to have more clarification around how the `filterText` works, with a few examples perhaps. Thx.
